packages/core/nuxt-theme-module/theme/layouts/default.vue

```diff
@@ -92,4 +92,10 @@ h4 {
   line-height: 1.6;
   margin: 0;
 }
+
+.app-footer {
```

Comment:
Imho we should put that in footer component because we need this in other layouts as well